core/domain/role_services.py

```diff
@@ -209,9 +208,10 @@ def get_role_graph_data():
     role_graph['links'] = []
     role_graph['nodes'] = {}
     for role in PARENT_ROLES:
-        role_graph['nodes'][role] = HUMAN_READABLE_ROLES[role]
-        for parent in PARENT_ROLES[role]:
-            role_graph['links'].append({'source': parent, 'target': role})
+        if role != feconf.ROLE_ID_SUPER_ADMIN:
```

Comment:
I think super admin shouldn't be part of the roles system at all, but an additional "outside" thing which allows a user to visit the admin page. It can be granted to a user with any role within the system, and it's managed outside the application itself. So I lean fairly strongly towards just removing it throughout -- if we need to reinstate it later, we can. Probably better to do that than leave things in a sort of "halfway state".